tools/flake8_plugin.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 import ast
-from typing import Any, Generator
+from typing import Any, Generator, List
```

Comment:
don't use `List`, use `list`

Reply:
we can, but in some parts of the codebase it breaks things -- notably anywhere `pydantic` has proliferated -- here it does not